pkg/status/status_otlp.go

```diff
@@ -8,24 +8,39 @@
 package status
 
 import (
-	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"fmt"
+	"io"
+	"net/http"
+
 	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/otlp"
 )
 
 // GetOTLPStatus parses the otlp pipeline and its collector info to be sent to the frontend
 func GetOTLPStatus() map[string]interface{} {
-	status := make(map[string]interface{})
-	otlpIsEnabled := otlp.IsEnabled(config.Datadog)
-	var otlpCollectorStatus otlp.CollectorStatus
-	if otlpIsEnabled {
-		otlpCollectorStatus = otlp.GetCollectorStatus(common.OTLP)
-	} else {
-		otlpCollectorStatus = otlp.CollectorStatus{Status: "Not running", ErrorMessage: ""}
+	if !otlp.IsEnabled(config.Datadog) {
+		return map[string]interface{}{
+			"otlpStatus":          false,
+			"otlpCollectorStatus": otlp.CollectorStatus{Status: "Not running"},
+		}
 	}
 
-	status["otlpStatus"] = otlpIsEnabled
-	status["otlpCollectorStatus"] = otlpCollectorStatus.Status
-	status["otlpCollectorStatusErr"] = otlpCollectorStatus.ErrorMessage
-	return status
+	var status, statuserr string
+	resp, err := getHTTPClient().Get(fmt.Sprintf("http://localhost:%s", config.Datadog.GetInt(config.OTLPHealthPort)))
+	if err != nil {
+		statuserr = fmt.Sprintf("Can not retrieve status: %s", err)
```

Comment:
nit: [Wrap the error](https://go.dev/blog/go1.13-errors#wrapping-errors-with-w) instead of stringifying in case we ever need to use `errors.Is`/`errors.As` 
```suggestion
		statuserr = fmt.Sprintf("Can not retrieve status: %w", err)
```